docs/batch_request.md

```diff
@@ -0,0 +1,24 @@
+## BatchRequest
+
+BatchRequest is useful when you want to make multiple requests efficiently. It batches all requests (upto 20 requests) into a json object and makes one api call. You can learn more about it on [Microsoft Docs](https://docs.microsoft.com/en-us/graph/json-batching). 
+
+## Code Sample
+
+```go
+reqInfo := abstractions.NewRequestInformation()
```

Comment:
I think that here we should take something from the fluent API like `client.Me().CreateGetRequestInformation()` for instance, to demonstrate it works with the fluent API.